code/modules/research/designs/autolathe_desings/autolathe_designs_sec_and_hacked.dm

```diff
@@ -73,46 +84,88 @@
 	build_path = /obj/item/restraints/handcuffs
 	category = list("hacked", "Security")
 
+/*
 /datum/design/receiver
 	name = "Modular Receiver"
 	id = "receiver"
 	build_type = AUTOLATHE | NO_PUBLIC_LATHE
 	materials = list(/datum/material/iron = 15000)
 	build_path = /obj/item/weaponcrafting/receiver
 	category = list("hacked", "Security")
+Old receiver, removed, you now use different receivers found in guncrafting.dm
+*/
+/datum/design/rifle_receiver
+	name = "Rifle Receiver"
+	id = "rifle_receiver"
+	build_type = AUTOLATHE
+	materials = list(/datum/material/iron = 40000)
```

Comment:
modify both this and the recipe to take 20 sheets of iron, reminder that I've nerfed material upgrade amounts, no need to balance by 0's anymore.